pages/createUpdate.js

```diff
@@ -127,12 +129,16 @@ class CreateUpdatePage extends React.Component {
   };
 
   render() {
-    const { data, LoggedInUser, intl } = this.props;
+    const { data, LoggedInUser, loadingLoggedInUser, intl } = this.props;
 
     if (!data.account) {
       return <ErrorPage data={data} />;
     }
 
+    if (loadingLoggedInUser) {
+      return <LoadingPlaceholder />;
```

Comment:
Does it really display something? You usually have to set a `height` for this to work

Reply:
Good point, here my intention was to show nothing until the logged user is loaded. For this we can just simplify by returning `null` instead. I've adjusted the code and as shown on the screen-cast above it shows nothing until the user is loaded. The other way I was thinking is to maybe show the `LoadingPlaceholder` on a part of the page, but this page returns the header and everything, and on the header itself we use the LoggedInUser so I thought it would be nice to just wait until it's loaded. Let me know if you think otherwise. 🤔

Reply:
It's always better to display a spinner or a loading frame rather than an empty space to let users know something's loading. Otherwise they may think the page just crashed

Reply:
Makes sense. I've added the `Loading` component so that it looks like this; 

![loading](https://user-images.githubusercontent.com/12435965/154089391-d77a6926-cb34-4ef7-8a9d-09884003d631.gif)